TestOutgoingTLSInvalidCertificate fails on Go1.20

```
$ go test -run TestOutgoingTLSInvalidCertificate
--- FAIL: TestOutgoingTLSInvalidCertificate (0.02s)
    client_test.go:1342: logged HTTP request * Request to https://127.0.0.1:42873
        > GET / HTTP/1.1
        > Host: example.com
        > User-Agent: Robot/0.1 crawler@example.com
        
        * tls: failed to verify certificate: x509: certificate signed by unknown authority
        ; want ^\* Request to https://127.0.0.1:42873
        > GET / HTTP/1\.1
        > Host: example\.com
        > User-Agent: Robot/0\.1 crawler@example\.com
        
        \* x509: .+
FAIL
```

Seems Go1.20 changes the error message again.

--- a/testdata/log.txtar
+++ b/testdata/log.txtar
@@ -670,7 +670,7 @@ Hello, world!
 > Host: example\.com
 > User-Agent: Robot/0\.1 crawler@example\.com
 
-\* x509: .+
+\* .*x509: .+
 -- TestOutgoingTooLongResponse --
 * Request to %s
 > GET /long-response HTTP/1.1

